docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -135,11 +135,20 @@ The `octopusdeploy/worker-tools` images provided for the execution containers fe
 
 The following worker-tools images are cached:
 
-- [Latest Windows-based image](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019): `!docker-image <octopusdeploy/worker-tools:windows.ltsc2019>`
+**On Linux Workers**:
 
 - [Latest Linux-based image](https://github.com/OctopusDeploy/WorkerTools/blob/master/ubuntu.18.04): `!docker-image <octopusdeploy/worker-tools:ubuntu.18.04>`
+- octopusdeploy/worker-tools:3.3.1-ubuntu.18.04
```

Comment:
I think we can still use the docker tags i.e. `!docker-image <octopusdeploy/worker-tools:3.3.1-ubuntu.18.04>` here if we want to keep the fonts consistent

Reply:
I think in DocSync, it's like, it will just ignore whatever the version you put there, and change it for the latest version?

Reply:
Sorry I was wrong, ignore this ⬆️